torch/quantization/observer.py

```diff
@@ -126,6 +143,8 @@ def observer(observer_cls, **kwargs):
     return partial(observer_cls, **kwargs)
 
 def default_observer(**kwargs):
+    # Restrict activations to be in the range (0,127)
```

Comment:
Wondering the reason for restricting the reduced precision on the activation instead of the weight? How will it affect the accuracy result if we instead restrict the range of weight (int8) to be (-64, 63)?

Reply:
From the overflow point of view, we have to reduce one bit for either one of them. I think the results would be similar if we reduce 1-bit for weights.

Reply:
I agree, I kept the weights at 8 bits because with QAT we can learn weights at a higher precision compared to 7. For post training quant, both choices lead to similar results.